src/main/scala/com/databricks/spark/csv/package.scala

```diff
@@ -38,6 +38,7 @@ package object csv {
         parserLib: String = "COMMONS",
         ignoreLeadingWhiteSpace: Boolean = false,
         ignoreTrailingWhiteSpace: Boolean = false,
+        treatEmptyValuesAsNulls: Boolean = false,
```

Comment:
/cc @marmbrus and @rxin as well, since I think this API design issue would be of interest to you.